index.html

```diff
@@ -352,6 +353,57 @@ <h2>Ballerina in Action</h2>
                   </div>
                </div>
             </div>
+            <div id="fourthTab" class="tab-pane fade in active"  >
+               <div class="row" style="background-color: #f5f5f5;">
+                  <div class="col-lg-12 col-md-12 col-sm-12 focusPane " style="padding:20px;">
+                     <div class="FullCode" style="background-color: #f5f5f5;">
+                        <div class="highlight"> 
+                           <pre class="language-ballerina basic ballerina">
+                              <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerina/grpc;
+
+listener grpc:Listener ep = new (9090);
+
+type ReadOnlyAlbum readonly & record {|
+      string id;
+      string title;
+      string artist;
+      float price;
+|};
+
+table<ReadOnlyAlbum> key(id) albums = table [
+         {id: "1", title: "Blue Train", artist: "John Coltrane", price: 56.99},
+         {id: "2", title: "Jeru", artist: "Gerry Mulligan", price: 17.99},
+         {id: "3", title: "Sarah Vaughan and Clifford Brown", artist: "Sarah Vaughan", price: 39.99}
+      ];
+
+@grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_RECORD_STORE, descMap: getDescriptorMapRecordStore()}
+service "RecordStore" on ep {
+
+      remote function getAlbumFromId(string id) returns Album|error {
+         ReadOnlyAlbum? album = albums[id];
+         if album is ReadOnlyAlbum {
+            return <Album>album;
+         }
+         return error grpc:NotFoundError(string `Related album for ID ${id} not found`);
+      }
+
+      remote function storeAlbum(Album album) returns Album|error {
+         albums.add(album.cloneReadOnly());
+         return album;
+      }
+
+      remote function getAlbums() returns stream<Album, error?>|error {
+         return albums.toArray().toStream();
+      }
+}                                 
+                              </code>
+                           </pre>
+                        </div>
+                     </div>
+                  </div>
+               </div>
+            </div>
          </div>
```

Comment:
are we going to add the proto def separately?

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-dev-website/pull/3692/commits/26c60765bd595915df9aac7ffc3d0cf7334e532f